components/Downloads/ChangelogModal/index.tsx

```diff
@@ -0,0 +1,43 @@
+import * as Dialog from '@radix-ui/react-dialog';
+import type { FC, PropsWithChildren, ReactNode, ComponentProps } from 'react';
+
+import AvatarGroup from '@/components/Common/AvatarGroup';
+// TODO: Add styling
+// import styles from './index.module.css';
```

Comment:
Just a FYI that once you do the styling it needs to be imported after all other imports